EmailService/EmailService.Data/SmtpSettingsRepository.cs

```diff
@@ -59,9 +59,14 @@ public async Task<bool> EditAsync(Guid moduleSettingId, JsonPatchDocument<DbModu
       return true;
     }
 
-    public async Task<DbModuleSetting> GetAsync()
+    public Task<DbModuleSetting> GetAsync()
     {
-      return await _provider.ModuleSettings.FirstOrDefaultAsync();
+      return _provider.ModuleSettings.FirstOrDefaultAsync();
+    }
+
+    public async Task<bool> DoesExistAnyAsync()
+    {
+      return await _provider.ModuleSettings.FirstAsync() is not null;
```

Comment:
you will get exception if module setting is empty

Reply:
changed to Any()